src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequest.java

```diff
@@ -10,9 +17,28 @@
     @JsonProperty("company_number")
     private final String companyNumber;
 
-    public ChipsRequest(String objectionId, String companyNumber) {
+    @JsonProperty("attachments")
+    private final Map<String, String> attachments;
+
+    @JsonProperty("reference_number")
+    private final String referenceNumber;
+
+    @JsonProperty("customer_email")
+    private final String customerEmail;
+
+    @JsonProperty("reason")
+    private final String reason;
+
+    public ChipsRequest(String objectionId, String companyNumber,
```

Comment:
I'd suggest putting each constructor param on a separate line to improve readability (easier to see at a glance how many params it takes and what they are).

Reply:
Ok on separate lines.